### PR TITLE
#23 - Sample code and comment style with Dummy code

### PR DESCRIPTION
**Pull request by HninPwint**

### CODE REVIEW

**Comment by MingMSDI** [comment]
One more thing, the team need to decide to use single quote or double quote on string

**Comment by HninPwint** [comment]
Yes, we can discuss in team's meeting today.

File: analysis/1/top5_country_suicide_rate_comparison_by_sex.ipynb [hunk 1]
```diff
@@ -95,7 +95,7 @@
    ],
    "source": [
     "## Get the countries names of top 5 highest suicide rate\n",
-    "print(top_10_suicide_rate_bothsex['Country'].to_list())\n",
+    "print(top_5_suicide_rate_bothsex['Country'].to_list())\n",
     "top_10_country_names = top_5_suicide_rate_bothsex['Country'].to_list()\n",
     "## Filtering out Male/Female records belongs to the hightest country.\n",
     "final = suicide_rate_separatesex[suicide_rate_separatesex['Country'].isin(top_10_country_names)]\n",
```
Comment (isampe) [inline analysis/1/top5_country_suicide_rate_comparison_by_sex.ipynb:101]: prefer to use a more readable variable name, instead of `final` ...

**Review by isampe — COMMENTED** [review]
On the section [3]

`suicide_rate_pivot_longer_df = pd.melt(suicide_rate_dataframe, id_vars=['Country', 'Sex'], value_vars=['80_above','70to79', '40to49', '30to39', '20to29', '10to19', 'all_age'], var_name='Age',value_name='Suicide_rate')`

is 60to69 column left intentionally?

File: analysis/1/top5_country_suicide_rate_comparison_by_sex.ipynb (hunk 1 above)
Comment (HninPwint) [reply analysis/1/top5_country_suicide_rate_comparison_by_sex.ipynb:101]: Thanks, Irfan.

**Pull request merged**